go.sum

```diff
@@ -251,8 +251,10 @@ github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/
 github.com/stretchr/testify v1.7.0 h1:nwc3DEeHmmLAfoZucVR881uASk0Mfjw8xYJ99tb5CcY=
 github.com/stretchr/testify v1.7.0/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 github.com/tozny/e3db-clients-go v0.0.159/go.mod h1:yNs03BRVrLCEZzRze0CjYkrMaC5SLIxPn0li1ZHHVW4=
-github.com/tozny/e3db-clients-go v0.0.198 h1:Zq+x30O568LYKRSIBrrUHShtLe//nQETObK5BvBEnVQ=
-github.com/tozny/e3db-clients-go v0.0.198/go.mod h1:yNs03BRVrLCEZzRze0CjYkrMaC5SLIxPn0li1ZHHVW4=
+github.com/tozny/e3db-clients-go v0.0.198-0.20211129172200-06b838d3685d h1:FyqPT86h+POERGMDgeAFZQd4YBdIKYuv92kBOyGirMw=
```

Comment:
`go mod tidy`

Reply:
THIS is the command. i tried running `go tidy` like 3 different times, obviously to no avail, because it is not a command. what i shoulda done is run `make lint` 😅